policies/bundles/CustomerKeyEncryption/metric.rego

```diff
@@ -4,13 +4,13 @@ default applicable = false
 
 default compliant = false
 
-keyUrl := input.atRestEncryption.keyUrl
+import input.atRestEncryption as enc
 
 applicable {
-	keyUrl != null
+	enc
```

Comment:
This is exemplary for all changes:
Just to be safe. In the case there is no enc.**keyUrl**, it is applicable but non-compliant. I think this is ok but just want to make sure that you also thought about that :)